### ISSUE TITLE
[Examples] COVID Vaccine dashboard contains query errors

### ISSUE THREAD

**Issue by octaviancorlade** [apache/superset#12528]
In a clean `master` or `latest` branch, with `docker-compose up` the COVID Vaccine dashboard contains many errors like this:

```
superset_app             | Traceback (most recent call last):
superset_app             |   File "/app/superset/connectors/sqla/models.py", line 1320, in query
superset_app             |     df = self.database.get_df(sql, self.schema, mutator)
superset_app             |   File "/app/superset/models/core.py", line 390, in get_df
superset_app             |     self.db_engine_spec.execute(cursor, sqls[-1])
superset_app             |   File "/app/superset/db_engine_specs/base.py", line 909, in execute
superset_app             |     cursor.execute(query)
superset_app             | psycopg2.errors.UndefinedColumn: column "stage_of_development" does not exist
superset_app             | LINE 2:            WHEN Stage_of_Development = 'Pre-clinical' THEN '...
superset_app             |                         ^
superset_app             | HINT:  Perhaps you meant to reference the column "covid_vaccines.Stage_of_Development".
```

This is expected with column names that are not all lowercase in Postgres, unless they are wrapped in double quotes

### Expected results

The example dashboards should work on a clean installation

### Actual results

The example COVID Vaccine dashboard contains many errors. The error in one of the charts is also displayed different than the others, but the problem seems to be the same in the database.

#### Screenshots

![image](https://user-images.githubusercontent.com/15089539/104635192-db114e80-56a1-11eb-81c1-7e7d8e908651.png)

#### How to reproduce the bug

```
git clone git@github.com:apache/superset.git
cd superset
docker-compose up
```
then login and open the dashboard

### Environment

(please complete the following information):

- superset version: Superset 0.999.0dev
- python version: Python 3.7.9
- node.js version: v12.20.1
- Docker version 20.10.2, build 2291f61
- docker-compose version 1.27.4, build 40524192

### Checklist

Make sure to follow these steps before submitting your issue - thank you!

- [X] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [X] I have reproduced the issue with at least the latest released version of superset.
- [X] I have checked the issue tracker for the same issue and I haven't found one similar.

**Comment by junlincc** [comment]
thanks for reporting! multiple users have reported, mistaking it as a functional issues. 
I consider this as a 1.0 release blocker. hopefully we can address is soon. 

@eschutho

**Comment by eschutho** [comment]
Thanks @junlincc for flagging. We're looking at this now.

**Comment by betodealmeida** [comment]
Working on it

**Comment by junlincc** [comment]
manually tested, confirmed issues is resolved in master, we will have it in for 1.0 release. thanks for the quick fix! @betodealmeida @eschutho
